### PR TITLE
[Redesign] Improve Lighthouse score

### PR DESCRIPTION
**Pull request by loic-sharma**
Lighthouse is a tool to check a website follows best practices: https://developers.google.com/web/tools/lighthouse

Category | Score before | Score after | Change
-- | -- | -- | --
Performance | 100 | 100 | -
Accessibility | 96 | 100 | +4
Best practices | 80 | 87 | +7
SEO | 91 | 100 | +9

Remaining issues I did not address:
* Fabric font loading inefficiently. May be fixed by updating `fabric.css`.
* Gallery assets served without efficient cache policy. Tracked by: https://github.com/NuGet/NuGetGallery/issues/8706
* Known vulnerabilities in jQuery@3.4.1 and Moment.js@2.18.1. Tracked by https://github.com/NuGet/Engineering/issues/3983
* SameSite cookie use by WCP consent cookie. Owned by WCP team

Addresses https://github.com/nuget/nugetgallery/issues/8605

### CODE REVIEW

File: src/NuGetGallery/Views/Packages/DisplayPackageV2.cshtml [hunk 1]
```diff
@@ -1056,7 +1060,7 @@
                                     <i class="ms-Icon ms-Icon--FabricFolderSearch" aria-hidden="true" aria-label="@disclaimer" title="@disclaimer"></i>
                                     <a href="@Model.NuGetPackageExplorerUrl" data-track="outbound-nugetpackageexplorer-url"
                                        aria-label="open in NuGet Package Explorer"
-                                       title="Explore additional package info on NuGet Package Explorer" target="_blank" rel="nofollow">
+                                       title="Explore additional package info on NuGet Package Explorer" target="_blank" rel="nofollow noreferrer">
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Packages/DisplayPackageV2.cshtml:1063]: This improves privacy and security.

See: https://stackoverflow.com/questions/50773152/when-should-i-use-rel-noreferrer
See: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel

`rel` value | Description
-- | --
nofollow | Indicates that the current document's original author or publisher does not endorse the referenced document.
noreferrer | No Referer header will be included. Additionally, has the same effect as noopener.
noopener | Creates a top-level browsing context that is not an auxiliary browsing context if the hyperlink would create either of those, to begin with (i.e., has an appropriate target attribute value).

File: src/NuGetGallery/Views/Packages/DisplayPackageV2.cshtml [hunk 2]
```diff
@@ -1113,17 +1115,17 @@
...
-                            {
-                                @ViewHelpers.AlertWarning(@<text>This package has no owners and is not being actively maintained.</text>)
-                            }
-                            else
-                            {
-                                foreach (var owner in Model.Owners)
+                        @if (!Model.Owners.Any())
+                        {
+                            @ViewHelpers.AlertWarning(@<text>This package has no owners and is not being actively maintained.</text>)
+                        }
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Packages/DisplayPackageV2.cshtml:1121]: Previously we would nest a `<div>` in a `<ul>` if the package had owners. That is not a best practice

File: src/NuGetGallery/Views/Packages/DisplayPackageV2.cshtml [hunk 3]
```diff
@@ -137,6 +137,10 @@
     }
 }
 
+@section Meta {
+    <meta name="description" content="@Model.ShortDescription">
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Packages/DisplayPackageV2.cshtml:141]: This improves our SEO

**Comment by zhhyu** [comment]
Looking now ...

**Review by dannyjdev — APPROVED** [review]

File: src/NuGetGallery/Views/Packages/DisplayPackageV2.cshtml (hunk 2 above)
Comment (zhhyu) [reply src/NuGetGallery/Views/Packages/DisplayPackageV2.cshtml:1121]: It should be "if the package had no owners", is it? Will the appearance be changed for packages without owners?

**Review by zhhyu — APPROVED** [review]

**Pull request merged**